app/models/bot/alegre.rb

```diff
@@ -123,56 +148,78 @@ def self.request_api(method, path, params = {})
     end
   end
 
-  def self.get_items_with_similar_title(pm, threshold)
-    self.get_items_with_similar_text(pm, 'title', threshold, pm.title)
+  def self.get_items_with_similar_title(pm, threshold, text_length_threshold=CONFIG["similarity_text_length_threshold"])
+    pm.title.to_s.split(" ").length > text_length_threshold ? self.get_items_with_similar_text(pm, 'title', threshold, pm.title) : {}
+  end
+
+  def self.get_items_with_similar_description(pm, threshold, text_length_threshold=CONFIG["similarity_text_length_threshold"])
+    pm.description.to_s.split(" ").length > text_length_threshold ? self.get_items_with_similar_text(pm, 'description', threshold, pm.description) : {}
```

Comment:
I think it's more reliable to split by `\s` than a single space.

Reply:
ok!